azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -263,8 +263,12 @@ public void setFlowLockErrorMessage(final String flowLockErrorMessage) {
     flowObj.put(SUBMITTIME_PARAM, this.submitTime);
 
     final List<Map<String, Object>> slaOptions = new ArrayList<>();
-    this.executionOptions.getSlaOptions().stream()
-        .forEach((slaOption) -> slaOptions.add(slaOption.toObject()));
+    List<SlaOption> slaOptionList = this.executionOptions.getSlaOptions();
+    if (slaOptionList != null) {
+      for (SlaOption slaOption : this.executionOptions.getSlaOptions()) {
```

Comment:
Nit: Use slaOptionList instead of using this.executionOptions.getSlaOptions() again.

Reply:
will do.